packages/react/src/components/Layout/Layout.tsx

```diff
@@ -173,9 +185,9 @@ class Layout extends UIComponent<ReactProps<LayoutProps>, any> {
     if (reducing && isSingleArea) {
       const composedClasses = cx(
         classes.root,
-        startArea && 'ui-layout--reduced__start',
-        mainArea && 'ui-layout--reduced__main',
-        endArea && 'ui-layout--reduced__end',
+        startArea && Layout.slotClassNames.reducedStart,
```

Comment:
to me it seems that there was a problem already there - the class we need to assign for reduced area is rather `'ui-layout__start--reduced`, not the `'ui-layout--reduced__start` that we've had. The reason is that, essentially, it is about marking the same `start` area, but using `reduced` state modifier for it